src/index.html

```diff
@@ -1,14 +1,65 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a class="header__logo" href="#home"><img alt="Logo" src="images/logo.png"></a>
+
+      <nav class="header__navigation nav">
```

Comment:
```suggestion
      <nav class="nav">
```

Reply:
You can remove class if you are not using it for styling

Reply:
AAAAAAA, like new block in block, thank you i am understand.